dev/bots/analyze.dart

```diff
@@ -472,6 +473,32 @@ Future<void> _verifyNoBadImportsInFlutterTools(String workingDirectory) async {
   }
 }
 
+final RegExp _licensePattern = RegExp(r'''Copyright \d{4}''');
+
+Future<void> _verifyNoMissingLicense(String workingDirectory) async {
+  final List<String> errors = <String>[];
+  for (FileSystemEntity entity in Directory(path.join(workingDirectory, 'packages'))
+      .listSync(recursive: true)
+      .where((FileSystemEntity entity) => entity is File && path.extension(entity.path) == '.dart')) {
+    final File file = entity;
+    bool hasLicense = false;
+    final List<String> lines = file.readAsLinesSync();
+    if (lines.isNotEmpty)
+      hasLicense = _licensePattern.firstMatch(lines.first) != null;
```

Comment:
Could this just be `lines.first.startsWith(_licensePattern)` with `_licensePattern = RegExp(r'// Copyright \d{4}''');`

Reply:
Done.